src/app/helpers/helpers.ts

```diff
@@ -290,6 +290,11 @@ export class Helpers extends HelpersBase {
         allowApiUpdate: Roles.lab,
         includeInApi: Roles.lab
     });
+    isIndependent = new BoolColumn({
```

Comment:
Same - I think that an admin should set these values - and not a dist center admin